analysers/analyser_merge_radio_support_FR.py

```diff
@@ -0,0 +1,192 @@
+                
+        Analyser_Merge.__init__(self, config, logger,
+            u"https://www.data.gouv.fr/datasets/551d4ff3c751df55da0cd89f",
+            u"Données sur les installations radioélectriques de plus de 5 watts",
+            CSV(Source(attribution = u"data.gouv.fr:ANFR", millesime = "08/2019",
+                    fileUrl = u"https://www.data.gouv.fr/fr/datasets/r/5da74526-7781-4726-b98b-232756643090", zip = "SUP_SUPPORT.txt", encoding = "ISO-8859-15"),
+                separator = u";", quote = u'$'),
+            Load(
+("CASE \"COR_CD_EW_LON\" WHEN 'W' THEN -1*(to_number(\"COR_NB_DG_LON\", '99') + to_number(\"COR_NB_MN_LON\", '99') / 60 + to_number(\"COR_NB_SC_LON\", '99') / 3600) WHEN 'E' THEN to_number(\"COR_NB_DG_LON\", '99') + to_number(\"COR_NB_MN_LON\", '99') / 60 + to_number(\"COR_NB_SC_LON\", '99') / 3600 END",), 
+("CASE \"COR_CD_NS_LAT\" WHEN 'S' THEN -1*(to_number(\"COR_NB_DG_LAT\", '99') + to_number(\"COR_NB_MN_LAT\", '99') / 60 + to_number(\"COR_NB_SC_LAT\", '99') / 3600) WHEN 'N' THEN to_number(\"COR_NB_DG_LAT\", '99') + to_number(\"COR_NB_MN_LAT\", '99') / 60 + to_number(\"COR_NB_SC_LAT\", '99') / 3600 END",),
+                  select = {"NAT_ID": NAT_IDs},
+                  uniq = ("SUP_ID",)
+                 ),
+            Mapping(
+                select = Select(
+                    types = ["nodes", "ways"],
+                    tags = tags_select),
+                conflationDistance = 50,
+                osmRef = "ref:FR:ANFR",
+                generate = Generate(
+                    static1 = tags_generate,
+                    static2 = {"source": self.source},
+                    mapping1 = {
+                        "ref:FR:ANFR": "SUP_ID",
+                        "operator": lambda fields: self.proprietaire[int(fields["TPO_ID"])] if fields["TPO_ID"] and int(fields["TPO_ID"]) in self.proprietaire else None,
+                        "height": lambda fields: fields["SUP_NM_HAUT"].replace(",", ".") if fields["SUP_NM_HAUT"] else None,
+                        },
+                    text = lambda tags, fields: {"en": u"%s, address : %s, %s%s" % ((lambda x: self.Tour_Mat_Pylone[fields["NAT_ID"]] if x == u"Tour, mât et pylône" else x)(title),
+                                            ", ".join(filter(lambda x: x != "None", [fields["ADR_LB_LIEU"], fields["ADR_LB_ADD1"], fields["ADR_LB_ADD2"], fields["ADR_LB_ADD3"],fields["ADR_NM_CP"]])),
+                                            (lambda x: self.communeNameIndexedByInsee[x] if x in self.communeNameIndexedByInsee else x)(fields["COM_CD_INSEE"]),
+                                            (lambda x: (u", operator : " + self.autre_proprietaire[int(x)]) if x and x != "None" and int(x) in self.autre_proprietaire else "")(fields["TPO_ID"])
+                                            )})))
+
+    # nombre : colonne TPO_ID dans SUP_SUPPORT.txt et valeur : SUP_PROPRIETAIRE.txt
+    proprietaire = {
```

Comment:
Can you switch the naming to english, please.

Reply:
J'ai traduit le commentaire et 2 variables.
Par contre pour "Tour_Mat_Pylone" que je retrouve dans le nom de l'analyse Analyser_Merge_Tour_Mat_Pylone pareil pour les autres noms d'analyses il y a des noms en français mais c'est les noms donnés dans la source de donnée.
redit moi si tu veux tout en anglais.

Reply:
ok